pkg/cortex/modules.go

```diff
@@ -341,6 +342,7 @@ func (t *Cortex) initIngester(cfg *Config) (err error) {
 	t.server.HTTP.Path("/ready").Handler(http.HandlerFunc(t.ingester.ReadinessHandler))
 	t.server.HTTP.Path("/flush").Handler(http.HandlerFunc(t.ingester.FlushHandler))
 	t.server.HTTP.Path("/shutdown").Handler(http.HandlerFunc(t.ingester.ShutdownHandler))
+	t.server.HTTP.Handle("/push", t.httpAuthMiddleware.Wrap(push.Handler(cfg.Distributor, t.ingester.Push)))
```

Comment:
Two things:
1. I'm a bit concerned about the inconsistency in the path between distributors (`/api/prom/push`) and ingesters (`/push`). Have you thought about it?
2. Could you document it to `docs/apis.md`, please?

Reply:
> I'm a bit concerned about the inconsistency in the path between distributors (/api/prom/push) and ingesters (/push). Have you thought about it?

Yeah, we can have them at the same path as the paths will collide in the single binary case.

I put it at `/push` to indicate its a private API; the /api/prom is for public endpoints.